src/doc/doc_phases/doc_html.nit

```diff
@@ -532,267 +464,142 @@ redef class DocComposite
 	end
 end
 
-redef class DocRoot
-
-	# Start the rendering from root.
-	#
-	# FIXME needed to maintain TplSection compatibility.
-	fun start_rendering(v: RenderHTMLPhase, doc: DocModel, page: MEntityPage): TplSection do
-		var section = new TplSection("top")
-		var mentity = page.mentity
-		section.title = mentity.html_name
-		section.subtitle = mentity.html_declaration
-		# FIXME ugly hack to avoid diff
+# FIXME hideous hacks to avoid diff
```

Comment:
When there is ugly hack it's to follow the current page organization and limit the diff.

It makes it easier to compare two successives versions of the doc. The removing of these hacks will be done in further PRs.